blog/_posts/2022-07-07-July-2022-update.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: "July 2022 Quarterly Update"
+description: ""
+tags: []
+---
+{% include JB/setup %}
+
+Dear GeoCAT Community,
+
+Please find below our updates for April, May, and June, 2022, including 
+team updates, developments, and community activities, since the last 
+quarterly update we made in early April:  
+
+
+### Project Raijin
+
+The collaboration between [Project Raijin](https://raijin.ucar.edu/) and 
+DOE's SEATS project has released the first official version, 
+[v2022.06.0](https://github.com/UXARRAY/uxarray/releases/tag/v.2022.6.0), of 
+the [UXarray](https://uxarray.readthedocs.io/en/latest/index.html) package. 
+Please see its [Conda distribution](https://anaconda.org/conda-forge/uxarray)
+page as well.
+
+Other than the release, there are other insights about the Project Raijin and 
+UXarray that we'd like to share:
+
+#### UXarray Discussions
+
+Having input and feedback from our community is vital for the success of the 
+UXarray package (so Project Raijin). We encourage our community members to share 
+their feedback with us and give input to the UXarray project lifecycle. In 
+accordance with this, we created a Github Discussion, [Prioritization of Uxarray 
+analysis operators](https://github.com/UXARRAY/uxarray/discussions/46), to listen 
+to our community's priorities with the data analysis functions to be implemented. 
+The discussion has received a lot of feedback and led to conversation between the 
+folks. We are looking forward to implementing such functions as well as to keep 

```

Comment:
```suggestion
community members. We are looking forward to implementing such functions as well as to keep 

```
This edit could also encompass deleting "the" from the ending of line 36 so the sentence would read "...to conversation between [the] community members"